modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaGetImagePreviewURLMethodCheck.testjava

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.document.library.kernel.util.DLUtil;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetImagePreviewURLMethodCheck {
+
+	public void method() {
+		imagemURL = DLUtil.getImagePreviewURL(
```

Comment:
Please rename variable to `imagePreviewURL`.

Reply:
Done!